src/app/app.cpp

```diff
@@ -817,9 +817,15 @@ bool CApplication::CreateVideoSurface()
     int vsync = 0;
     if (GetConfigFile().GetIntProperty("Experimental", "VSync", vsync))
     {
-        SDL_GL_SetSwapInterval(vsync);
+        if (SDL_GL_SetSwapInterval(vsync) == -1)
+        {
+            GetLogger()->Warn("Adaptive sync not supported.\n");
```

Comment:
This could also be called when setting something else than adaptive sync and it's not supported